plugins/module_utils/foreman_helper.py

```diff
@@ -655,10 +690,237 @@ def exit_json(self, changed=False, **kwargs):
         super(ForemanAnsibleModule, self).exit_json(**kwargs)
 
 
-class ForemanEntityAnsibleModule(ForemanAnsibleModule):
+class ResolverEntityMixin(object):
```

Comment:
This mixin is only used in `ForemanEntityAnsibleModule`, so I wonder if we should just merge it into that class, which would also probably solve our import test issue?

Reply:
Yes, I wanted to keep it as a Mixin at least at start, to avoid possible merge conflicts, but that make sense to merge it, and yes it solve import problems.